### PR TITLE
Bench 146 caching package dependencies

### PR DESCRIPTION
**Pull request by JoeCSykes**

### CODE REVIEW

**Review by NogaAD — APPROVED** [review]

**Comment by abbas-khan8** [comment]
Do we need the `poetry.lock` file as part of the workflow?

File: .gitignore [hunk 1]
```diff
@@ -144,6 +144,3 @@ GitHub.sublime-settings
 .idea/*
 *.sqlite3
 
-
-# Poetry
-poetry.lock
```
Comment (abbas-khan8) [inline .gitignore:149]: Why are we removing this?

**Review by MichaelOSullivanAnswer — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (JoeCSykes) [reply .gitignore:149]: the poetry lock file is needed by actions/setup-python@v4 to cach the dependencies

Comment (JoeCSykes) [reply .gitignore:149]: see the error message here https://github.com/AnswerConsulting/AnswerKing-Python/actions/runs/3429246367

Comment (PietroConvalleAD) [reply .gitignore:149]: it should be generated during install, maybe because we run `poetry update` rather than `poetry install`

Comment (JoeCSykes) [reply .gitignore:149]: running poetry install causes it to error

**Pull request merged**